client/src/components/HealthCareProvider/HCProviderDashboard.tsx

```diff
@@ -1,28 +1,78 @@
-                    {/* <img className="w-24 h-24 mb-3 rounded-full shadow-lg" src="/healthprovider.jpg" alt="Health provider image" />*/}
-                    <h5 className="mb-1 text-xl font-medium text-gray-900 dark:text-white"> {user?.name}</h5>
-                    <span className="text-sm text-gray-500 dark:text-gray-400">{user?.email}</span>
-                    <div className="flex mt-4 space-x-3 md:mt-6">
-                        <button className="inline-flex items-center px-3 py-2 text-sm font-medium text-center text-white bg-blue-700 rounded-lg hover:bg-blue-800 focus:ring-4 focus:outline-none focus:ring-blue-300 dark:bg-blue-600 dark:hover:bg-blue-700 dark:focus:ring-blue-800" onClick={() => logouthandler()}>Logout</button>
-                    </div>
-                </div>
-            </div>
+        <div className="flex flex-col justify-center items-center">
+        <div className="w-full max-w-sm bg-white border border-gray-200 rounded-lg shadow-md dark:bg-gray-800 dark:border-gray-700 flex flex-col items-center mt-0">
+          <h5 className="mt-4 mb-1 text-xl font-medium text-gray-900 dark:text-white">{user?.name}</h5>
+          <span className="mb-4 text-sm text-gray-500 dark:text-gray-400">{user?.email}</span>
+          <button className="mb-4 inline-flex items-center px-3 py-2 text-sm font-medium text-center text-white bg-blue-700 rounded-lg hover:bg-blue-800 focus:ring-4 focus:outline-none focus:ring-blue-300 dark:bg-blue-600 dark:hover:bg-blue-700 dark:focus:ring-blue-800" onClick={() => logouthandler()}>Logout</button>
+        </div>
+        <div className="mt-8 w-full max-w-screen-lg mx-auto">
+          <table className="w-full">
+            <thead>
+              <tr>
+                <th className="px-6 py-3 text-center">Doctor Name</th>
+                <th className="px-6 py-3 text-center">Credentials Hash</th>
+                <th className="px-6 py-3 text-center">Status</th>
+                <th className="px-6 py-3 text-center">Action</th>
+              </tr>
+            </thead>
+            <tbody className="divide-y divide-gray-200">
+              {verificationRequests && verificationRequests.map((request: { doctorName: string | number | boolean | React.ReactElement<any, string | React.JSXElementConstructor<any>> | React.ReactFragment | React.ReactPortal | null | undefined; credentialsHash: string | number | boolean | React.ReactElement<any, string | React.JSXElementConstructor<any>> | React.ReactFragment | React.ReactPortal | null | undefined; approved: any; }, index) => (
+                <tr key={index}>
+                  <td className="px-6 py-4 text-center">{request.doctorName}</td>
+                  <td className="px-6 py-4 text-center">{request.credentialsHash}</td>
+                  <td className="px-6 py-4 text-center">{request.approved ? "Approved" : "Pending"}</td>
+                  <td className="px-6 py-4 text-center">
+                    {!request.approved && (
```

Comment:
if request.approved is false, there is no distinction between pending and rejected requests. this seems to be a bug.